apps/src/StudioApp.js

```diff
@@ -1269,7 +1294,7 @@ StudioApp.prototype.onMouseMoveVizResizeBar = function (event) {
  * Resize the visualization to the given width
  */
 StudioApp.prototype.resizeVisualization = function (width) {
-  var codeWorkspace = document.getElementById('codeWorkspace');
+  var workspaceRight = $(".workspace-right");
```

Comment:
Wonder if there's a better name than workspace-right? (Especially given that in RTL langs it might be the left side?). Basically this concept is meant to cover the pane that isn't the visualization. Right now it's called codeWorkspace even though it includes the debugger. This class is used to align any panes/items that are to the right of the visualization.